### PR TITLE
DM-46425: Move DECam precursor step to its own pipeline

### PR DESCRIPTION
**Pull request by leeskelvin**

### CODE REVIEW

**Review by fred3m — APPROVED** [review]
LGTM. I'm surprised that this is the best way to replace a task, but if this is what's required then I'm fine with it. You should probably add a comment in the description that mentions this ticket and the the ticket that will make it obsolete (or at least the conditions for reverting it).

File: pipelines/DECam/DRP-Merian.yaml [hunk 1]
```diff
@@ -1,6 +1,12 @@
...
+  Prior to running subsets or tasks in this pipeline, the DECam prerequisite
+  task isrForCrosstalkSources must be run. More information on that task can be
+  found in the isrForCrosstalkSources.yaml pipeline file.
 imports:
-  - $DRP_PIPE_DIR/pipelines/_ingredients/DECam/DRP.yaml
-  - $ANALYSIS_DRP_DIR/pipelines/analysis_drp_plots.yaml
+  - location: $DRP_PIPE_DIR/pipelines/_ingredients/DECam/DRP.yaml#isrForCrosstalkSources
+    exclude:
+      - isrForCrosstalkSources
+  - location: $ANALYSIS_DRP_DIR/pipelines/analysis_drp_plots.yaml
```
[the remark this answers is not in the document]
Comment (leeskelvin) [reply pipelines/DECam/DRP-Merian.yaml:12]: Right, I didn't want to touch the main `_ingredients` file, as long-term the hope is that this short-term workaround will not be required. However, I don't have a feel for how long that long-term fix will be in place, so we needed something that works for now.

My solution here was to leave `_ingredients` as-is, but exclude the offending task from the user-facing pipeline YAML, and construct a second YAML containing only the offending task.

However, I'm not averse to doing something completely different if you can think of a more optimal approach?

[the remark this answers is not in the document]
Comment (leeskelvin) [reply pipelines/DECam/DRP-Merian.yaml:12]: The alternative to the approach on this PR (lets call this PR option a) is to strip `isrForCrosstalkSources` out of the main `_ingredients` file and put it either b) into its own `_ingredients` file, which then gets imported into its own user-facing YAML, or c) just define the task directly in its own user-facing YAML.

I don't have a strong preference

[the remark this answers is not in the document]
Comment (mrawls) [reply pipelines/DECam/DRP-Merian.yaml:12]: I agree this is weird, but I think if you add an in-line comment explaining why you import a task and then immediately exclude it, all will be forgiven 😄

[the remark this answers is not in the document]
Comment (leeskelvin) [reply pipelines/DECam/DRP-Merian.yaml:12]: Ah, wait! Apologies, I did not intend to push the `#isrForCrosstalkSources` - this is a mistake. The power of review for the win.

It should instead just be the main pipeline YAML.

[the remark this answers is not in the document]
Comment (mrawls) [reply pipelines/DECam/DRP-Merian.yaml]: That makes more sense!!

[the remark this answers is not in the document]
Comment (leeskelvin) [reply pipelines/DECam/DRP-Merian.yaml]: Thanks for picking up on this @fred3m and @mrawls! :smiling_face_with_tear:

**Pull request merged**